dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -537,29 +537,34 @@ RtpsUdpDataLink::leave_multicast_group(const NetworkInterface& nic)
 
 void
 RtpsUdpDataLink::add_locators(const RepoId& remote_id,
-                              const ACE_INET_Addr& narrow_address,
-                              const ACE_INET_Addr& wide_address,
+                              const AddrSet& narrow_addresses,
+                              const AddrSet& wide_addresses,
                               bool requires_inline_qos)
 {
-  if (narrow_address == ACE_INET_Addr()) {
-    ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) ERROR: RtpsUdpDataLink::add_locators: narrow_address for %C is empty\n"), LogGuid(remote_id).c_str()));
-    return;
+  if (narrow_addresses.empty()) {
+    ACE_ERROR((LM_WARNING, ACE_TEXT("(%P|%t) WARNING: RtpsUdpDataLink::add_locators: narrow_addresses for %C is empty\n"), LogGuid(remote_id).c_str()));
```

Comment:
error or warning?  also on 549.

If it ends up as a warning, should there be an error if both sets are empty?

Reply:
I'm thinking an error if both are empty and that's it.